File: bharath46/TwoSum.java
class TwoSum {
    public int[] twoSum(int[] nums, int target) 
    {
        HashMap<Integer, Integer> map = new HashMap<>();

        for(int i = 0; i< nums.length; i++)
        {
            int com = target - nums[i];
            if(map.containsKey(com))
            {
                return new int[] { map.get(com), i};
            }

            map.put(nums[i],i);
        }
        throw new IllegalArgumentException("no two sum");
    }
}

File: Jacklin/twosum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        h = {} 
        for i, n in enumerate(nums): 
            c = target - n  
            if c in h: 
                return [h[c], i]
            h[n] = i
